android/src/main/java/com/getcapacitor/community/speechrecognition/SpeechRecognition.java

```diff
@@ -276,9 +276,6 @@ private void beginListening(
             speechRecognizer.setRecognitionListener(listener);
             speechRecognizer.startListening(intent);
             SpeechRecognition.this.listening(true);
-            if (partialResults) {
```

Comment:
Just realized: I guess, this is a breaking change, if someone expects the Promise to resolve immediately. 

But I would still consider it to be worth it.